### PR TITLE
Update color palette

### PR DESCRIPTION
**Pull request by ltclmbdev**

### CODE REVIEW

File: src/tachyons-colors.css [hunk 1]
```diff
@@ -13,90 +13,111 @@
...
+  --white-40: rgba(255, 255, 255, .4);
+  --white-30: rgba(255, 255, 255, .3);
+  --white-20: rgba(255, 255, 255, .2);
+  --white-10: rgba(255, 255, 255, .1);
+  --white-05: rgba(255, 255, 255, .05);
+  --white-025: rgba(255, 255, 255, .025);
+  --white-0125: rgba(255, 255, 255, .0125);
+  
+  
+  /* Previous Vars */
```
Comment (joelhooks) [inline src/tachyons-colors.css]: This section can go.

File: src/tachyons-colors.css [hunk 2]
```diff
@@ -13,90 +13,111 @@
...
- --blue: #009dcf;
- --tag-blue: #edf0ff;
- --dark-green: #3ab66e;
- --green: #add960;
- --turquoise: #42c2b3;
- --tag-turquoise: #d9f3f0;
- --dark-red: #8f1233;
- --red: #e13f58;
+  --black: #000;
+  --white: #fff;
```
Comment (joelhooks) [inline src/tachyons-colors.css:17]: Vojta listed an alternative for white as `F4F5F9`

File: src/tachyons-colors.css [hunk 3]
```diff
@@ -13,90 +13,111 @@
...
+  --light-navy: var(--dark-blue);
+  --dark-gray: var(--white-10);
+  --gray: var(--white-30);
+  --light-gray: var(--white-secondary);
+  --tag-gray: var(--white-90);
+  --dark-orange: var(--orange-secondary);
+  --tag-orange: var(--white-70);
+  --tag-blue: var(--white-70);
+  --dark-green: var(--green-secondary);
+  --green: var(--green-secondary);
```
Comment (joelhooks) [inline src/tachyons-colors.css]: Vojta has this listed as `59CD90`

File: src/tachyons-colors.css [hunk 4]
```diff
@@ -13,90 +13,111 @@
...
+  
+  
+  
+  /* Still active previous vars */
+  
+  --dark-navy: var(--base-secondary);
+  --navy: var(--base);
+  --light-navy: var(--dark-blue);
+  --dark-gray: var(--white-10);
+  --gray: var(--white-30);
```
Comment (joelhooks) [inline src/tachyons-colors.css]: Listed as `63768D`

File: src/tachyons-colors.css [hunk 5]
```diff
@@ -13,90 +13,111 @@
...
+  --white-secondary: #f8f8f8;
+  
+  --base: #232c3b;
+  --base-secondary: #171e27;
+  
+  /*--gray: #f4f5f9;*/
+  --gray-secondary: #e2e3e7;
+  
+  /*--dark-gray: #63768d;*/
+  --dark-gray-secondary: #8996a5;
```
Comment (joelhooks) [inline src/tachyons-colors.css]: no dark gray was listed in updated colors.

**Review by joelhooks — CHANGES_REQUESTED** [review]
```
--base: #131922 ;
  --base-secondary: #090C11 ;

  --white: #F4F5F9 ;
  --white-secondary: #E2E3E7 ;

  --gray: #63768D ;
  --gray-secondary: #B0B6BE ;

  --dark-blue: #344055 ;
  --dark-blue-secondary: #1C222C ;

  --blue: #4786FF ;
  --blue-secondary: #386FDA ;

  --green: #59CD90 ;
  --green-secondary: #409B6B ;

  --orange: #FD9126 ;
  --orange-secondary: #C97420 ;

  --red: #F0624D ;
  --red-secondary: #C44C3A ;

  --yellow: #FBC155 ;
  --yellow-secondary: #D2A043 ;

   --dark-navy: var(--base-secondary);
   --navy: var(--base);
   --light-navy: var(--dark-blue);
   --dark-gray: var(--white-10);
   --gray: var(--white-30);
   --light-gray: var(--white-secondary);
   --tag-gray: var(--white-90);
   --dark-orange: var(--orange-secondary);
   --tag-orange: var(--white-70);
   --tag-blue: var(--white-70);
   --dark-green: var(--green-secondary);
   --green: var(--green-secondary);
   --turquoise: var(--green-secondary);
   --tag-turquoise: var(--white-70);
   --dark-red: var(--red-secondary);
```

please double check against the list of values provided by @vojtaholik above.

File: src/tachyons-colors.css (hunk 5 above)
Comment (ltclmbdev) [reply src/tachyons-colors.css]: http://cloud.egghead.io/jdWi

Comment (joelhooks) [reply src/tachyons-colors.css]: ah, he updated it! Looks good .

**Review by joelhooks — APPROVED** [review]

**Pull request merged**